app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -229,6 +230,13 @@ open class FenixApplication : LocaleAwareApplication() {
         // no-op, LeakCanary is disabled by default
     }
 
+    // This is for issue https://github.com/mozilla-mobile/fenix/issues/11660. We prefetch our info for startup
+    // so that we're sure that we have all the data available as our fragment is launched.
+    private fun prefetchForHomeFragment() {
+        StrictMode.allowThreadDiskReads().resetPoliciesAfter {
```

Comment:
Is this necessary? I don't see anywhere that would cause us to read on the main thread but perhaps I misunderstand.

Reply:
Not sure why, but our strictmodes complained when I didn't add it and the CI tests failed ( some of them). I could try to remove it